src/pages/Patient/VerifyPatient.tsx

```
@@ -101,6 +102,14 @@ export default function VerifyPatient() {
                       ],
                     });
                   }}
+                  children={
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)